app/routes/content/index.js

```diff
@@ -85,7 +86,9 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
             .then(([provider, node, license]) => {
                 const title = node.get('title');
                 const description = node.get('description');
-                const doi = preprint.get('doi');
+                const mintDoi = extractDoiFromString(preprint.get('links.preprint_doi'));
+                const peerDoi = preprint.get('doi');
+                const doi = peerDoi !== ''? peerDoi: mintDoi;
```

Comment:
Please add whitespace on both sides of ? and :

See: http://eslint.org/docs/rules/space-infix-ops